### ISSUE TITLE
Request: support Android 11 (R - API 31) 

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [pranavpandey/dynamic-toasts#10]
Sadly on Android R, you can't set a custom View for a Toast (when targeting API 31) :

https://developer.android.com/reference/android/widget/Toast#setView(android.view.View)
https://developer.android.com/reference/android/widget/Toast#getView()

Please offer something that mimics the standard Toast's look&feel, so that at least we could have it shown within the app, as a normal View.
Maybe something similar to what we have when we long press the action items on the toolbar:

[dfu6CwFaPM.zip](https://github.com/GrenderG/Toasty/files/5610631/dfu6CwFaPM.zip)

Custom Toasts are useful anyway usually only when the user is inside the app...

**Comment by pranavpandey** [comment]
Yes, I know about that and I am using a [different approach](https://github.com/pranavpandey/dynamic-support/tree/master/dynamic-support/src/main/java/com/pranavpandey/android/dynamic/support/widget/tooltip) to show tooltips in my apps. It is based on the default `app compat` implementation but it needs a `view` to attach, so it would not be a complete replacement but still it covers most of the use cases.

I will try to implement something similar (more generalized) in the future updates.
